package.json

```diff
@@ -33,5 +33,9 @@
     "npm": ">=7.0.0",
     "node": ">=14.0.0"
   },
-  "packageManager": "npm@8.10.0"
+  "packageManager": "npm@8.10.0",
+  "dependencies": {
+    "@citation-js/core": "^0.6.8",
+    "@citation-js/plugin-csl": "^0.6.9"
+  }
```

Comment:
We should only have to add these in the sub-package, not at the top!

Reply:
Wait, how did this happen?! I think my editor must have done this. I didn't notice the path when I was being lazy and using update for the add strategy. Yikes!